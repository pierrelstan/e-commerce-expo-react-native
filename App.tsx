import * as React from "react";
import { ThemeProvider } from "styled-components/native";
// eslint-disable-next-line @typescript-eslint/no-unused-vars
import { ApolloProvider, useQuery, gql } from "@apollo/client";
import client from "./AppolloConfiguration";
import StackNavigation from "./Navigations/Stack/stack";
import { theme } from "./theme";

// const GET_LOCATIONS = gql`
//   query GetLocations {
//     locations {
//       id
//       name
//       description
//       photo
//     }
//   }
// `;
// function DisplayLocations() {
//   const { loading, error, data } = useQuery(GET_LOCATIONS);

//   if (loading) return <Text>Loading...</Text>;
//   if (error) {
//     return <Text>Error :{error.message}</Text>;
//   }

//   return data.locations.map(({ id, name, description, photo }: any) => (
//     <View key={id}>
//       <Text>{name}</Text>
//       <Image alt="location-reference" source={{ uri: `${photo}` }} />

//       <Text>About this location:</Text>
//       <Text>{description}</Text>
//     </View>
//   ));
// }

export default function App() {
  return (
    <ApolloProvider client={client}>
      <ThemeProvider theme={theme}>
        <StackNavigation />
      </ThemeProvider>
    </ApolloProvider>
  );
}

/* <Container>
        <Text>E-commerce stanley</Text>
        <StatusBar />
        <DisplayLocations />
      </Container> */

// const Container = styled.View`
//   flex: 1;
//   background-color: #fff;
//   align-items: center;
//   justify-content: center;
// `;
// const View = styled.View``;
// const Image = styled.Image`
//   width: 100px;
//   height: 200px;
// `;

// const Text = styled.Text``;
// const StatusBar = styled.StatusBar``;
