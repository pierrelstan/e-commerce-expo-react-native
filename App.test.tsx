import * as React from "react";
import { render, screen } from "@testing-library/react-native";

// import { render, fireEvent } from "./test-utils";
import App from "./App";

test("examples of some things", () => {
  render(<App />);

  expect(screen.toJSON()).toMatchSnapshot();
});
