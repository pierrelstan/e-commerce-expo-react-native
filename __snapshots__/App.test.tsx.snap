// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`examples of some things 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  s
  <Text
    style={
      [
        {},
      ]
    }
  >
    E-commerce
  </Text>
</View>
`;
